my_exe/test/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 # test dependencies
-include(CTest)
+# TODO(CK): Not needed! include(CTest)
```

Comment:
It should be removed if it is not needed

Reply:
done